forge/src/main/java/org/openjdk/skara/forge/gitlab/GitLabMergeRequest.java

```diff
@@ -781,6 +781,22 @@ public void makeNotDraft() {
         }
     }
 
+    @Override
+    public Optional<ZonedDateTime> lastMarkedAsDraftTime() {
+        var draftMessage = "marked this merge request as **draft**";
+        var notes = request.get("notes").execute();
+        var lastMarkedAsDraftTime = notes.stream()
+                .map(JSONValue::asObject)
+                .filter(obj -> obj.get("system").asBoolean())
+                .filter(obj -> draftMessage.equals(obj.get("body").asString()))
+                .reduce((a, b) -> a)
```

Comment:
Same here as for GitHub, maybe reduce to highest date just to be safe?

Reply:
Same as above and also adjust it.